diff --git a/dist/accounts/android-native-account.js b/dist/accounts/android-native-account.js
index 315cd5714101c1899c7d07309cbcca6d80d31e65..8a37fb46dc0ce2ae6791fc73ca85c823bb9a97d0 100644
--- a/dist/accounts/android-native-account.js
+++ b/dist/accounts/android-native-account.js
@@ -14,7 +14,8 @@ export class AndroidNativeAccount extends BaseAccount {
     }
     static fromJSON(json) {
         const signer = new AndroidNativeSigner(json.signer.packageName);
-        return new AndroidNativeAccount(json.pubkey, signer);
+        const account = new AndroidNativeAccount(json.pubkey, signer);
+        return super.loadCommonFields(account, json);
     }
     static async fromApp(app) {
         const signer = new AndroidNativeSigner(app.packageName);
